README.md

```diff
@@ -108,3 +89,22 @@ Support us with a monthly donation and help us continue our activities.
 Become a sponsor to get your logo on our README on Github.
 
 [![Sponsors](https://opencollective.com/spyder/sponsors.svg)](https://opencollective.com/spyder#support)
+
+
+## More information
```

Comment:
This project is completely independent of Spdyer, so please remove this section.

Reply:
I'm not sure we share a mutual understanding of the phrase "completely independent", considering it is owned by the Spyder IDE Github organization, is developed by the same people, and Spyder is its original motivating development use case... 

In any case, removed as requested.

Reply:
Yeah, I meant that even if we develop it, it's misleading to tell people that they're going to find more information about it in Spyder related websites because that's basically not true.

Thanks for the quick response, by the way :+1:

Reply:
Right, that makes a lot of sense. Good thinking.

Of course! Responding to and addressing feedback on my existing PRs is my highest priority (my internal task priority order being: Responding to PR feedback > Reviewing/testing requested PRs > Responding to user comments on issue threads > Working on priority/blocking issues/tasks > Triaging issues > Working on assigned issues/tasks > Reviewing other PRs > Unassigned work items > Other tasks)